<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Centroid</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1054</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_ReadData"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_ImageViewer"/>
   <addaction name="action_ParticlesViewer"/>
   <addaction name="separator"/>
   <addaction name="action_PlayFrames"/>
   <addaction name="action_ReadData"/>
   <addaction name="action_PreviousFrame"/>
   <addaction name="action_NextFrame"/>
   <addaction name="action_ZoomOut"/>
   <addaction name="action_ZoomIn"/>
   <addaction name="action_ScrollHandDrag"/>
   <addaction name="action_SelectionMode"/>
  </widget>
  <widget class="QDockWidget" name="dockModulesWidget">
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>160</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame { border: 1px solid gray; border-radius: 0px; padding: 0px; }</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Data Processing Modules</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockModulesWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayoutModulesDockWidget">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="indentation">
          <number>0</number>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockLogsWidget">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>111</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Centroid Logs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockListWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalListLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="plainLogsTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_ReadData">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Read Data</string>
   </property>
   <property name="statusTip">
    <string>Read Data</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_ImageViewer">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/icons/view.png</normaloff>:/icons/view.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Image Viewer</string>
   </property>
   <property name="toolTip">
    <string>Image Viewer</string>
   </property>
  </action>
  <action name="action_PreviousFrame">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/icons/back.png</normaloff>:/icons/back.png</iconset>
   </property>
   <property name="text">
    <string>Previous Frame</string>
   </property>
   <property name="toolTip">
    <string>Previous Frame</string>
   </property>
  </action>
  <action name="action_NextFrame">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/icons/forward.png</normaloff>:/icons/forward.png</iconset>
   </property>
   <property name="text">
    <string>Next Frame</string>
   </property>
  </action>
  <action name="action_PlayFrames">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Play Frames</string>
   </property>
  </action>
  <action name="action_ZoomIn">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/icons/zoom-out.png</normaloff>:/icons/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="action_ZoomOut">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/icons/zoom-in.png</normaloff>:/icons/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="action_ScrollHandDrag">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/icons/hand.png</normaloff>:/icons/hand.png</iconset>
   </property>
   <property name="text">
    <string>Scroll Hand Drag</string>
   </property>
  </action>
  <action name="action_SelectionMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/icons/select.png</normaloff>:/icons/select.png</iconset>
   </property>
   <property name="text">
    <string>Selection Mode</string>
   </property>
  </action>
  <action name="action_ParticlesViewer">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/icons/particles.png</normaloff>:/icons/particles.png</iconset>
   </property>
   <property name="text">
    <string>Particles Viewer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="main_window.qrc"/>
 </resources>
 <connections/>
</ui>
